### ISSUE TITLE
Command shortage on menu

### ISSUE THREAD

**Issue by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#1900]
Raised by @marmotian on forum https://forum.sumatrapdfreader.org/t/actions-only-available-through-keyboard/3749

There is a noticeable lack of Next/Previous file accessibility via the menu, thus cannot be accessed via touchscreen / tablet users.

< Ctrl > + < Shift > + Right | open next document in the directory
-- | --
< Ctrl > + < Shift > + Left | open previous document in the directory

Unsure where they could be best placed as File menu is busy but GoTo (next file / previous file) seems just as appropriate

The request was to **either or also** include as icons on Toolbar

**Comment by Marmotian** [comment]
Just to clarify, I think that these commands should be included on a menu somewhere regardless. The "Go To" menu seems like a reasonable place.

I think the function is used frequently enough to justify toolbar buttons.
